<?xml version="1.0"?>
<sdf version="1.6">
  <model name="basket_blue">
    <static>true</static>
    <link name="link">
      <collision name="collision">
        <geometry>
          <mesh>
            <uri>model://basket_blue/meshes/basket.obj</uri>
            <scale>0.003 0.0015 0.0015</scale>
          </mesh>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://basket_blue/meshes/basket.obj</uri>
            <scale>0.003 0.0015 0.0015</scale>
          </mesh>
        </geometry>
        <material>
          <ambient>0.0 0.0 1.0 1</ambient>
          <diffuse>0.0 0.0 1.0 1</diffuse>
          <specular>0.2 0.2 0.2 1</specular>
        </material>
      </visual>
    </link>
  </model>
</sdf>
